.github/workflows/release.yml

```diff
@@ -159,14 +169,31 @@ jobs:
         source ./sourceme.sh && optic_build_for_release
     - run: echo "//registry.npmjs.org/:_authToken=${{ secrets.PUBLISH_NPM_OPTIC_BOT }}" > ~/.npmrc
     - run: node ./workspaces/scripts/publish.js
+      if: false
       env:
         OPTIC_PUBLISH_SCOPE: public
+    - name: Set is_prerelease output
+      id: release_type
+      run: |
+        OPTIC_VERSION=$(node -e "console.log(require('./workspaces/local-cli/package.json').version)")
+        if [[ $OPTIC_VERSION == *'-'* ]]; then
+          echo "v$OPTIC_VERSION is a prerelease"
+          echo "::set-output name=is_prerelease::true"
+        else
+          echo "v$OPTIC_VERSION is not a prerelease"
+          echo "::set-output name=is_prerelease::false"
+        fi
 
   # Creates debian package for users to install, hosted on s3
   release-deb:
     runs-on: ubuntu-latest
     needs: release-npm
+    if: needs.release-npm.outputs.is_prerelease != 'true'
     steps:
+      - run: |
+          echo "determined as not a prerelease" 
```

Comment:
similar to the above this reads to me as something that doesn't need negation. "determined to be a release" vs. "determined as not a prerelease"

Reply:
Added in a 4 or 5 push debug cycle, will clean up!

Reply:
Actually, not changing, as this will just be deleted as part of the debug code when we get rid of the `exit 1` safety.